python/ray/data/dataset.py

```diff
@@ -328,8 +330,12 @@ def map(
                 tasks, ``ray.data.ActorPoolStrategy(size=n)`` to use a fixed-size actor
                 pool, or ``ray.data.ActorPoolStrategy(min_size=m, max_size=n)`` for an
                 autoscaling actor pool.
+            num_cpus: The number of CPUs to reserve for each parallel map worker.
```

Comment:
What does the default value (`None`) do?

Reply:
Defaults to Ray Core